docker-compose.override.services.yml

```diff
@@ -0,0 +1,94 @@
+services:
+
+  smtp4dev:
+    image: rnwood/smtp4dev:v3
+    restart: always
+    ports:
+      # Web interface
+      - ${SMTP4DEV_WEB_PORT}:80
+      # SMTP server
+      - ${SMTP4DEV_SMTP_PORT}:25
+      # IMAP
+      - ${SMTP4DEV_IMAP_PORT}:143
+    volumes:
+        - smtp4dev_data:/smtp4dev
+    environment:
+      # Specifies the server hostname. Used in auto-generated TLS certificate if enabled.
+      - ServerOptions__HostName=smtp4dev
+
+  db:
+    image: postgis/postgis:13-3.1-alpine
+    restart: unless-stopped
+    environment:
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    ports:
+      - ${HOST_POSTGRES_PORT}:5432
+    command: ["postgres", "-c", "log_statement=all", "-c", "log_destination=stderr"]
+
+  geodb:
+    image: postgis/postgis:12-3.0
```

Comment:
Should we use the same image for db and geodb? This would avoid pulling in two images (for a deprecated service)

Reply:
Good spot! Adjusted